Projects/hCoreUI/Sources/PresentationStyle+Detented.swift

```diff
@@ -63,12 +63,15 @@ class DetentedTransitioningDelegate: NSObject, UIViewControllerTransitioningDele
     var options: PresentationOptions
     var wantsGrabber: Bool
     var viewController: UIViewController
-    let bag = DisposeBag()
     var keyboardFrame: CGRect = .zero
+    var cancellables = Set<AnyCancellable>()
 
     func listenToKeyboardFrame() {
-        bag += viewController.view.keyboardSignal(priority: .highest)
-            .onValue { [weak self] event in guard let self = self else { return }
+
+        viewController.view.keyboardSignal(priority: .highest)
```

Comment:
keyboardSignal is still using FLOW. Let me know if you need any help with it

I think that here instead of
`var cancellables = Set<AnyCancellable>()`

we should use 
`var keyboardCancellable: AnyCancellable?
`
like 
`keyboardCancellable = viewController.view.keyboardSignal etc
`

Reply:
updated to var keyboardCancellable: AnyCancellable?

Reply:
but still need to fix keyboardSignal to not use flow